Sources/TestSupport/misc.swift

```diff
@@ -72,11 +72,12 @@ public func fixture(name: String, tags: [String] = [], file: StaticString = #fil
                 try systemQuietly([Git.tool, "-C", dstDir.asString, "init"])
                 try systemQuietly([Git.tool, "-C", dstDir.asString, "config", "user.email", "example@example.com"])
                 try systemQuietly([Git.tool, "-C", dstDir.asString, "config", "user.name", "Example Example"])
+                try systemQuietly([Git.tool, "-C", dstDir.asString, "config", "commit.gpgsign", "false"])
                 try systemQuietly([Git.tool, "-C", dstDir.asString, "add", "."])
                 try systemQuietly([Git.tool, "-C", dstDir.asString, "commit", "-m", "msg"])
```

Comment:
We should use GitRepository here since we have it now, can you do that or just put a FIXME here that we should move get rid of these calls